src/webpage/themes.css

```diff
@@ -1,178 +1,176 @@
-	--reply-bg: #d4d6e9;
-	--reply-text: #2e2e30;
-	--spoiler-hover: #b9b9b9;
-	--spoiler-open-bg: #dadada;
-	--unknown-file-bg: #bdbdbd;
-	--unknown-file-border: #adadad;
-	--login-border: #c3c0e0;
-	--loading-bg: #b5b7cc;
-	--dialog-bg: #c1c8d6;
-	--dialog-border: #b9b7db;
-	--scrollbar-track: #d5d1e2;
-	--scrollbar-thumb: #b0afc0;
-	--scrollbar-thumb-hover: #a5a5b8;
-	--markdown-timestamp: #c8c8da;
-	--embed: #f2f3f5;
-	--link: #3333ee;
-	--discovery-bg: #c6c6d8;
-	--message-jump:#ccceff;
-	--icon-color:black;
+
+	--primary-bg: #fefefe;
+	--primary-hover: #f6f6f9;
+	--primary-text: #4b4b59;
+	--primary-text-soft: #656575;
+
+	--secondary-bg: #e0e0ea;
+	--secondary-hover: #d0d0dd;
+
+	--servers-bg: #b4b4ca;
+	--channels-bg: #eaeaf0;
+	--channel-selected: #c7c7d9;
+	--typebox-bg: #ededf4;
+
+	--button-bg: #cacad8;
+	--button-hover: #b3b3c4;
+	--spoiler-bg: #dadada;
+	--link: #056cd9;
+
+	--black: #4b4b59;
+	--green: #68d79d;
+	--primary-text-prominent: #08080d;
+	--secondary-text: #3c3c46;
+	--secondary-text-soft: #4c4c5a;
+	--dock-bg: #d1d1df;
+	--dock-hover: #b8b8d0;
 }
+
 .Light-theme {
 	color-scheme: light;
 
-	--primary-text: #000;
-	--primary-bg: #8e90c3;
-	--black: #fff;
-	--shadow: #000;
-	--focus: #5e50c5;
-
-	--message-bg-hover: #5757b5;
-	--typing-bg: #d4d6e9;
-	--profile-bg: #8075bf;
-	--profile-info-bg: #8075bf;
-	--timestamp-color: #000000;
-	--code-bg: #a89adf;
-	--info-bg: #6060a3;
-	--user-info-bg: #796f9a;
-	--user-dock-bg: #83839d;
-	--channels-bg: #c2c2d1;
-	--channel-hover: #726e88;
-	--blank-bg: #5e50c5;
-	--light-border: #000000;
-	--settings-hover: #b5b1bb;
-	--quote-bg: #7a798e;
-	--button-bg: #5757b5;
-	--button-hover: #8e90c3;
-	--textarea-bg: #abb1cd;
-	--filename: #47bbff;
-	--mention-bg: #F00;
-	--mention-md-bg: #3b588b;
-	--pronouns: #202020;
-	--channel-name-bg: #c0c0d4;
-	--server-name-bg: #a3a3b5;
-
-	--server-border: #aaaac4;
-	--server-hover: #7f7fa8;
-
-	--reply-border: #474b76;
-	--reply-bg: #d4d6e9;
-	--reply-text: #38383d;
-
-	--spoiler-hover: #34333a;
-	--spoiler-open-bg: #767587;
-
-	--unknown-file-bg: #bdbdbd;
-	--unknown-file-border: #adadad;
-
-	--login-border: #c3c0e0;
-	--loading-bg: #b5b7cc;
-
-	--dialog-bg: #c1c8d6;
-	--dialog-border: #b9b7db;
-
-	--scrollbar-track: #d2cedf;
-	--scrollbar-thumb: #bdbcca;
-	--scrollbar-thumb-hover: #a7a7be;
-	--markdown-timestamp: #c8c8da;
-	--embed: #cdccd1;
-	--link: #5566cc;
-	--discovery-bg: #c6c6d8;
-	--message-jump:#ccceff;
-	--icon-color:black;
+	--primary-bg: #aaafce;
+	--primary-hover: #b1b6d4;
+	--primary-text: #060415;
+	--primary-text-soft: #424268;
+
+	--secondary-bg: #9397bd;
+	--secondary-hover: #9ea5cc;
+	
+	--servers-bg: #7a7aaa;
+	--channels-bg: #babdd2;
+	--channel-selected: #9c9fbf;
+	--typebox-bg: #bac0df;
+
+	--button-bg: #babdd2;
+	--button-hover: #9c9fbf;
+	--spoiler-bg: #34333a;
+	--link: #283c8b;
+
+	--black: #434392;
+	--red: #ca304d;
+	--secondary-text-soft: #211f2e;
+	--blank-bg: #494985;
+	--spoiler-text: #e4e6ed;
+}
+
+.Accent-theme {
```

Comment:
Why is this separated out like this?

Reply:
See previous but I wanted to still have the tint functionality

Reply:
I meant why not have it in the themes like it was?

Reply:
This way it was possible to set just the accent(color splash) without setting the tint(affect whole theme) at the same time
Maybe we could make it separate variables?

Reply:
I meant more so that the themes might want to handle it instead, as the defaults might not agree with the theme, like if it were too light or too dark for a certain theme

Reply:
Oh, accent-theme is a separate dark theme that happens to mix the accent color with everything, the way the previous Dark-theme did. It could be named Dark-accent-theme

Reply:
_Oh_ I thought it was a separate style for for the accent colors, yeah, defiantly needs a rename then lol